middle_end/flambda2/types/reify.ml

```diff
@@ -72,7 +72,7 @@ let try_to_reify_fields env ~var_allowed alloc_mode
               | Tagged_immediate _imm -> Some simple
               | Naked_immediate _ | Naked_float _ | Naked_float32 _
               | Naked_int32 _ | Naked_vec128 _ | Naked_int64 _
-              | Naked_nativeint _ ->
+              | Naked_nativeint _ | Null ->
```

Comment:
I think that's wrong, but because this whole `const` case is wrong (independently of Null cases). I will submit a fix separately.

Reply:
(It's in #3206)